test/e2e/logging_test.go

```diff
@@ -5,84 +5,78 @@ import (
 
 	operatorsv1 "github.com/openshift/api/operator/v1"
 	deploymentsub "github.com/openshift/console-operator/pkg/console/subresource/deployment"
-	"github.com/openshift/console-operator/pkg/testframework"
+	"github.com/openshift/console-operator/test/e2e/framework"
 )
 
-func setupLoggingTestCase(t *testing.T) *testframework.Clientset {
-	client := testframework.MustNewClientset(t, nil)
-	testframework.MustManageConsole(t, client)
-	testframework.MustNormalLogLevel(t, client)
-	return client
+func setupLoggingTestCase(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
 }
 
-func cleanUpLoggingTestCase(t *testing.T, client *testframework.Clientset) {
-	testframework.WaitForSettledState(t, client)
+func cleanUpLoggingTestCase(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
 }
 
 // TestDebugLogLevel sets 'Debug' LogLevel on the console operator and tests
 // if '--log-level=*=DEBUG' flag is set on the console deployment
 func TestDebugLogLevel(t *testing.T) {
-	client := setupLoggingTestCase(t)
-	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	client, _ := setupLoggingTestCase(t)
+	defer cleanUpLoggingTestCase(t, client)
 
-	err := testframework.SetLogLevel(t, client, operatorsv1.Debug)
+	err := framework.SetLogLevel(t, client, operatorsv1.Debug)
 	if err != nil {
 		t.Fatalf("error: %s", err)
 	}
 
-	deployment, err := testframework.GetConsoleDeployment(client)
+	deployment, err := framework.GetConsoleDeployment(client)
 	if err != nil {
 		t.Fatalf("error: %s", err)
 	}
 	flagToTest := deploymentsub.GetLogLevelFlag(operatorsv1.Debug)
 	if !isFlagInCommand(t, deployment.Spec.Template.Spec.Containers[0].Command, flagToTest) {
 		t.Fatalf("error: flag (%s) not found in command %v \n", flagToTest, deployment.Spec.Template.Spec.Containers[0].Command)
 	}
-	cleanUpLoggingTestCase(t, client)
+
 }
 
 // TestTraceLogLevel sets 'Trace' LogLevel on the console operator and tests
 // if '--log-level=*=TRACE' flag is set on the console deployment
 func TestTraceLogLevel(t *testing.T) {
-	client := setupLoggingTestCase(t)
-	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
-
-	err := testframework.SetLogLevel(t, client, operatorsv1.Trace)
+	client, _ := setupLoggingTestCase(t)
```

Comment:
The same comment as above

Reply:
interim atm, see above https://github.com/openshift/console-operator/pull/248/files#r307366710